docs/architecture/deep-dives/passkeys/credentials.md

```diff
@@ -0,0 +1,114 @@
+---
+sidebar_position: 2
+---
+
+# Credentials
+
+This page provides an in-depth look at the FIDO2 credential.
+
+## What is a Credential?
```

Comment:
⛏️ All titles don't repeat capitalization; remove the capital C here for example. Apply everywhere.